src/common/file_location.c

```diff
@@ -146,7 +146,33 @@ char *dt_loc_find_install_dir(const char *suffix, const char *searchname)
   g_free(curr);
   return g_strdup(tmp);
 }
+#elif defined(_WIN32)
+char *dt_loc_find_install_dir(const char *suffix, const char *searchname)
+{
+  gchar *runtime_prefix;
+  gchar *slash;  
+  gchar *finaldir;
+  wchar_t fn[PATH_MAX];
+  
+  GetModuleFileNameW (NULL, fn, G_N_ELEMENTS (fn));
```

Comment:
does not matter too much, but since you asked, the `.clang-format` should have formatted that as `GetModuleFileNameW(NULL, fn, G_N_ELEMENTS(fn));`, i.e. without spaces before the brace.